plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/form/form.tsx

```diff
@@ -150,6 +175,54 @@ const Form = < T extends AddressFormValues | ContactFormValues >( {
 					);
 				}
 
+				if ( field.key === 'address_2' ) {
+					return (
+						<Fragment key={ field.key }>
```

Comment:
We're using a keyed fragment here, as the fragment contains conditional logic. Using the shorthand fragment <> does not allow us to add the key and would trigger a unique "key" prop warning.